### PR TITLE
feat-nav

### PR DESCRIPTION
**Pull request by iamsaurav28**
Screenshot (76)

### CODE REVIEW

**Review by vaibhav-mahalle — APPROVED** [review]
No apparent changes in the code. Clean and nicely indented code. Keep up the good practices

File: comp-layouts/nav.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+<head>
+     <meta charset="UTF-8">
+     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/stylesheets/components/nav.css">
+  <link rel="stylesheet" href="/stylesheets/main.css">
+     <title>Document</title>
+</head>
+<body>
+     <div class="wrapper">
```
Comment (krishanu2911) [inline comp-layouts/nav.html]: you can make the class name as nav-wrap it is easy to understood

File: stylesheets/components/nav.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  height: 80px;
+  width: 100%;
+  background: var(--navbar-bg);
+}
+
+.logo {
+  position: relative;
+  left: 40px;
+  top: 20px;
+  font-size: xx-large;
```
Comment (krishanu2911) [inline stylesheets/components/nav.css:18]: just a suggestion, not mandatory change xx-large could be xx-lg is short and easy to understand

**Review by krishanu2911 — COMMENTED** [review]
Well structured code and just added a few suggestions else seems good to go!!

File: stylesheets/components/nav.css (hunk 2 above)
Comment (iamsaurav28) [reply stylesheets/components/nav.css:18]: lol. bro how can i change that its inbuilt code already

File: comp-layouts/nav.html (hunk 1 above)
Comment (iamsaurav28) [reply comp-layouts/nav.html]: cool done

**Review by krishanu2911 — APPROVED** [review]

**Pull request merged**